zad3/app/views/bestiary/novigrad_monsters.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1>BestiaryController#novigrad_monsters</h1>
-<p>Find me in app/views/bestiary_controller/novigrad_monsters.html.erb</p>
+<table>
```

Comment:
Hmmm, ten widok jest łudząco podobny do `bestiary/index.html.erb` - żeby nie duplikować kodu można skorzystać z [partiali](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials)

Reply:
No zgadza się, jak w drugim widoku nie usuwałem lokacji, to tym bardziej powinienem tak zrobić, dzięki :)

Reply:
No zgadza się, jak w drugim widoku nie usuwałem lokacji, to tym bardziej powinienem tak zrobić, dzięki :)